tools/stress-cluster/chaos/examples/network-stress-example/templates/testjob.yaml

```diff
@@ -12,6 +12,6 @@ spec:
     - name: network-example
       command: ["bash", "poll.sh"]
       imagePullPolicy: Always
-      image: {{ .Values.image }}
+      image: {{ .Stress.Scenario.image }}
```

Comment:
Do we need to initialize an empty scenario object in addons for cases where it's not specified?

Reply:
I think it should just fail if that's the case